### ISSUE TITLE
`kneaddata 0.10.0` produces only unpaired reads

### ISSUE THREAD

**Issue by valentynbez** [microbiome-immunity-project/metagenomome_assembly#24]
1. Paired end files remain unmatched by kneaddata
2. BioBakery did not adress the problem https://github.com/biobakery/kneaddata/issues/26
3. Maybe a `bowtie2 2.2.5` problem, which is installed by default  
![image](https://user-images.githubusercontent.com/61702053/151037967-6538e90b-d4c1-4431-bf8f-3ceae7819e14.png)
 https://forum.biobakery.org/t/problem-with-paired-end-demo-on-new-install/1004/6

**Comment by tvatanen** [comment]
Using `--reorder` argument for kneaddata should help (if the paired and files are "matches" to begin with) 

` --reorder             order the sequences in the same order as the input`

**Comment by valentynbez** [comment]
- added the `--reorder` parameter, attaching script for sample processing from Cromwell
[script.txt](https://github.com/microbiome-immunity-project/metagenomome_assembly/files/7940259/script.txt)
- run `kneaddata` on 49 paired samples from [Mongolian gut study](https://www.ncbi.nlm.nih.gov/Traces/study/?query_key=1&WebEnv=MCID_61f10e6a7ba6e1213b721c4d&o=acc_s%3Aa) - no paired reads detected, which imo highly improbable.
![image](https://user-images.githubusercontent.com/61702053/151133716-928cdcbc-b8d9-4805-b853-05909d7092df.png)

**Comment by valentynbez** [comment]
- bumped `bowtie2` to the latest version `2.4.5`
- uncovered `kneaddata` DEBUG messages
```01/26/2022 01:03:27 PM - kneaddata.utilities - DEBUG: b'67503593 reads; of these:\n 67503593 (100.00%) were unpaired; of these:\n 67425905 (99.88%) aligned 0 times\n 42460 (0.06%) aligned exactly 1 time\n 35228 (0.05%) aligned >1 times```

I guess having high proportion of unpaired reads in many samples is possible then. 
Not sure if the issue title makes sense now.

**Comment by valentynbez** [comment]
Found explanation on https://forum.biobakery.org/t/paired-end-wgs-sequence-analysis/426/7, looks like a false alarm on my side.

**Comment by valentynbez** [comment]
Not a false alarm, the problem was with our data. We used the data downloaded form `SRA`. As `kneaddata` uses FASTQ headers, the data from `SRA` had completely wrong headers, not compatible with FASTQ schema. https://forum.biobakery.org/t/does-kneaddata-0-7-4-still-require-the-1-and-2-in-the-read-ids/1577/2
I bumped up `kneaddata` to a development version, where `input1` and `input2` could be specified. It all works now.
